frontend/src/pages/CourseSelector/courseMenu/CourseMenu.less

```diff
@@ -32,6 +32,10 @@
   font-weight: 700;
 }
 
+.locked {
```

Comment:
Personally, I think font-weight 100 looks so thin that it affects legibility. But if we do increase it, I think some experimentation may be needed here to maintain a visual difference between unlocked, available and selected courses. Fe team can tinker around with this later.